src/utils/tests/validation.test.js

```diff
@@ -172,3 +173,110 @@ describe('validation.js - validateVmName', () => {
     expect(validateVmLikeEntityName('vm1', vmSettings, props)).toEqual(getValidationObject(VIRTUAL_MACHINE_EXISTS));
   });
 });
+
+describe('validation.js - isValidMAC', () => {
+  // octets with colon separator
+  it('returns true for a valid 6 octet MAC address with colon separator', () => {
```

Comment:
This would look more readable IMO:


```js
const testConfig = [{
  title: 'returns true for a valid 6 octet MAC address with colon separator',
  mac: '01:23:45:67:89:ab',
}]
testConfig.forEach(c => {
  it(c.title,  () => {
    expect(isValidMAC(c.mac)).toBeTruthy()
  }
})
```

you can make two configs for truthy/falsy results or indicate the result in the config itself

Reply:
I did this, but mainly to be consistent with how other tests are written. I don't find it more readable than having an explicit test for each scenario, as the tests are written to be read like a sentence. With the config setup the reader has to mentally map the contents of each object to the function. So, I think the config setup is better in terms of conciseness, but inferior in terms of readability.

As an aside, in the original version, I think it would have been better for me to call the function directly in the `expect()` call instead of storing the return value to a variable. That would have saved several lines.

Reply:
I guess it is subjective, but it is more readable to me. I can see all the testcases in one places and don't have to trawl through other unrelated lines.

Reply:
Agreed. It's totally subjective. I'm marking this as resolved since the change has been made.